server/src/test/java/org/elasticsearch/index/IndexVersionTests.java

```diff
@@ -193,8 +190,6 @@ public void testParseLenient() {
         }
     }
 
-    @UpdateForV9(owner = UpdateForV9.Owner.CORE_INFRA)
-    @AwaitsFix(bugUrl = "can be unmuted once lucene is bumped to version 10")
```

Comment:
No update needed given that we still support reading from v7 indices, hence Lucene 8 is a known major version (what `getLowestReadCompatibleVersion.luceneVersion().major` returns).